### PR TITLE
add full dust recipe in plat-line

### PR DESCRIPTION
**Pull request by GlodBlock**
add full dust recipe for rest part in platline

### CODE REVIEW

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java [hunk 1]
```diff
@@ -98,18 +98,22 @@ private static void runHelperrecipes() {
         for (Werkstoff w : Werkstoff.werkstoffHashSet)
             if (w.containsStuff(Materials.Sulfur) && (w.containsStuff(Materials.Copper) || w.containsStuff(Materials.Nickel))) {
                 GT_Values.RA.addChemicalRecipe(w.get(crushedPurified), GT_Utility.getIntegratedCircuit(1), AquaRegia.getFluidOrGas(150), PTConcentrate.getFluidOrGas(150), null, 250);
-                GT_Values.RA.addChemicalRecipe(w.get(crushedPurified), PTMetallicPowder.get(dust), AquaRegia.getFluidOrGas(1150), PTConcentrate.getFluidOrGas(1150), PTResidue.get(dustTiny), 250);
+                GT_Values.RA.addChemicalRecipe(w.get(crushedPurified), GT_Utility.getIntegratedCircuit(9), AquaRegia.getFluidOrGas(1350), PTConcentrate.getFluidOrGas(1350), null, 2250);
```
Comment (Skiftcha) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java]: w.get(crushedPurified, 9) ?

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java [hunk 2]
```diff
@@ -98,18 +98,22 @@ private static void runHelperrecipes() {
...
+                GT_Values.RA.addChemicalRecipe(w.get(crushedPurified, 9), PTMetallicPowder.get(dust, 9), AquaRegia.getFluidOrGas(10350), PTConcentrate.getFluidOrGas(10350), PTResidue.get(dust), 2250);
```
Comment (Skiftcha) [inline src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java:102]: old recipe Werkstoff + PTMetallicPowder + AquaRegia removed with no backward compatibility?

**Review by Skiftcha — COMMENTED** [review]
something strange or i dont get the logic

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java (hunk 1 above)
Comment (GlodBlock) [reply src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java]: fixed

File: src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java (hunk 2 above)
Comment (GlodBlock) [reply src/main/java/com/github/bartimaeusnek/bartworks/system/material/GT_Enhancement/PlatinumSludgeOverHaul.java:102]: necessary break. ;) bascially few people use crushed ore to get extra platium in their platline. it won't cause much problem but can solve a huge ass in packing

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**